<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SeqPasterWidget</class>
 <widget class="QWidget" name="SeqPasterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>303</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create annotation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Paste data here</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="U2::SequenceTextEdit" name="sequenceEdit">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Custom settings</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Alphabet:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="alphabetBox"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QRadioButton" name="skipRB">
        <property name="text">
         <string>Skip unknown symbols</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QRadioButton" name="replaceRB">
        <property name="text">
         <string>Replace unknown symbols with</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="symbolToReplaceEdit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maxLength">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>U2::SequenceTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header location="global">U2Gui/SequenceTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>skipRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>symbolToReplaceEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>replaceRB</sender>
   <signal>toggled(bool)</signal>
   <receiver>symbolToReplaceEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
